packages/nodejs/src/transmitter.ts

```diff
@@ -63,14 +62,12 @@ export class Transmitter {
   }
 
   private requestOptions(requestUrl: URL): HashMap<any> {
-    const configData = this.#config.data
-
     return {
       method: "POST",
       protocol: requestUrl.protocol,
       host: requestUrl.hostname,
       port: requestUrl.port,
-      path: requestUrl.toString(),
+      path: requestUrl.pathname + requestUrl.search,
```

Comment:
Yes, that's the format needed to add proxy support. But I think this change is good as this is the correct usage of the `path` option when you're not using a proxy. Both ways work anyway.

I also think it'll be a good idea to change the HTTP client to Axios once we add proxy support to avoid future complications.

Reply:
It will need to do that when running it through a proxy, as that's what HTTP proxies expect -- but it's not supposed to do that when _not_ running through a proxy. It seems like many HTTP servers, in practice, deal with it fine: both our Sinatra mock server in the diagnose tests, and the actual server handling diagnose reports, are okay with it. It's still not the correct behaviour, I don't think, and `nock` wasn't having it.